README.fh.md

```diff
@@ -21,9 +19,54 @@ git remote add origin https://github.com/ledgerwatch/erigon.git
 git fetch origin
 ```
 
-##### Assumptions
+### Assumptions
 
 For the best result when working with this repository and the scripts it contains:
 
 - The remote `sf` exists on main module and points to `git@github.com:ChainSafe/erigon.git`
 - The remote `origin` exists on main module and points to https://github.com/ledgerwatch/erigon.git
+
+### Install Erigon
+
+To install Erigon, run the below command
+```
+go install ./cmd/erigon
+```
+
+### Running battlefield test
+
+[Battlefield](https://github.com/streamingfast/battlefield-ethereum) requires geth and Erigon for the test. Clone the battlefield repo, install the clients(geth and Erigon) and run the below command to run the test
+```
+cd battlefield-ethereum
+./bin/compare_vs_oracle.sh erigon
+```
+
+### Running Erigon
+
+Erigon can be used as an Execution Layer (EL) for Consensus Layer clients (CL). Default configuration is OK. 
+
+If your CL client is on a different device, add --authrpc.addr 0.0.0.0 (Engine API listens on localhost by default) as well as --authrpc.vhosts <CL host>.
+
+Starting Erigon for goerli chain
+```
+erigon --datadir=/datadir/erigon/data   --firehose-enabled --chain=goerli --externalcl --private.api.addr=localhost:9090 --http.api=eth,erigon,web3,net,debug,trace,txpool,parity
```

Comment:
Instead of `--datadir=/datadir/erigon/data`, we can specify `--datadir=<path_to_datadir>` something like that

Reply:
fixed